Ozone Project/Ozone/src/test/java/Ozone.java

```diff
@@ -0,0 +1,118 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ozone {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    String filePath = "D:/DiscountedElements.txt";
+    boolean hasNextPage = true;
```

Comment:
You assume that you are going to have a next page, either change the cycle or change the name of the var.